lighthouse-core/report/html/renderer/util.js

```diff
@@ -72,6 +72,21 @@ class Util {
       }
     }
 
+    if (clone.stackPacks) {
```

Comment:
If we go that direction I would probably argue there's not much difference just adding it to the audit JSON directly at that point :)

The biggest reason to have them as a standalone property in the LHR (`lhr.stackPacks`) IMO is the icon and overall link/description. Given that we need a place for those things to go keyed by stackPack id/name, it makes sense to nest their audit descriptions beneath them rather than keep duplicating the stackPack id/name.

Reply:
having a separate entry in LHR makes LHR much smaller like @patrickhulce suggested but @exterkamp you're the LHR guru and I don't really have an opinion on this

Reply:
I'm a fan of separate toplevel entry in LHR. And by pack first.  E.g. `lhr.stackPacks[0].descriptions.unused-css-rules`